sw/device/silicon_creator/manuf/tests/BUILD

```diff
@@ -260,6 +260,7 @@ cc_library(
         srcs = ["sram_empty_functest.c"],
         cw310 = cw310_jtag_params(
             bitstream = ":bitstream_rom_exec_disabled_{}".format(lc_state.lower()),
+            test_cmd = "--elf={firmware}",
```

Comment:
Is there a problem being solved by relocating this here?

Reply:
Yes, previous you put in the openocd test commands but some tests like the manufacturing tests do not accept an `--elf` option and instead bootstrap a binary. For jtag tests, I think each test has to specify how they want to provide the firmware file, I don't see a way to unifty this.

Reply:
[This](https://github.com/lowRISC/opentitan/pull/19958/files#diff-07875233cbf1931b9341dacff929573d2a8e9c3c3c3c6ca3a14a03280fd0c30aR212) is an example.

Reply:
There is also the case where a test harness loads two different elfs over jtag, one after the other: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/manuf/skus/earlgrey_a0/generic/BUILD;drc=7eb94909c1a3166f4f220b05d55f09b81ddca689;l=133